### DISCUSSION TITLE
Open tabs bar upon opening link in new tab for compact mode

### DISCUSSION THREAD

**Discussion by MordechaiHadad** [zen-browser/desktop#3464] [Ideas]
Currently in compact mode, whenever you create a new tab with `CTRL+T`, it will open the sidebar, but when you open a link in a new tab by clicking on the mousewheel upon a link, it will not open it, which doesn't indicate that it actually opened it. So for that reason, when you open a link in a new tab, something like this or another indicator would be very helpful.

**Comment by vervamond** [comment]
Yeah, I could've sworn this was already a thing, but since this is an old issue, I might just be imagining the past.
